tests/RobotFramework/tests/cumulocity/service_monitoring/service_monitoring.robot

```diff
@@ -99,26 +99,6 @@ Check health status of tedge-mapper-c8y service on broker restart
 
     Custom Test Teardown
 
-Check health status of child device service
```

Comment:
Why was this test removed and not just adapted to use the new health endpoints? I understand that this test probably was relying on the `tedge_to_te_converter` earlier and that's why you removed. But, we could have just updated it to use the new `te/.../health/status` endpoints directly, unless this is duplicate and a test for child device service health status already exists elsewhere.

Reply:
I may have made a mistake with that one. We agreed with @reubenmiller that health status messages should not change entity properties, so it follows that they shouldn't also create entities by themselves either.
However I forgot about auto-registration mechanism, in which it is expected that an entity sending any message will be automatically registered in C8y, so the same goes for health status messages.

I'll try to restore and update the test.